scripts/ui-test-hostname-setup.sh

```diff
@@ -2,7 +2,7 @@
 # This script sets the ip of the addons-frontend image as localhost within the selenium-firefox image.
 # This MUST be run before any user integration tests.
 
-UI_IP="`docker inspect addons-server_addons-frontend_1 | grep "IPAddress" | grep -oE "\b([0-9]{1,3}\.){3}[0-9]{1,3}\b"`"
+UI_IP="`docker inspect addonsserver_addons-frontend_1 | grep "IPAddress" | grep -oE "\b([0-9]{1,3}\.){3}[0-9]{1,3}\b"`"
```

Comment:
This is changed because I noticed it was different from my local config, so we now install docker-compose using curl, and a local tester should be able to use this script as well.